### DISCUSSION TITLE
title & file problems

### DISCUSSION THREAD

**Discussion by sonnaad** [aclap-dev/video-downloadhelper#177] [Help and Q&A]
### Your question

Hi, when I download a file from some sites it's not even the title of the video or the extension it's supposed to be. I end up needing to rename or add the extension when it's done downloading. IDK what happened it just started doing it I never had issues with it before I used smart naming rules and it didn't do anything/

### Addon & CoApp version and details

VDH-Version 8.2.0.23 (stable)
Firefox-122

**Comment by paulrouget** [comment]
> VDH-Version 8.2.0.23 (stable)

8.2.1 fixes this issue. It will autoupdate soon.
